forum/qa-plugin/report-reason/script.js

```diff
@@ -0,0 +1,95 @@
+$(document).ready(function()
+{
+    $(".qa-form-light-button-flag").attr("type", "button");
+    
+    $(".qa-form-light-button-flag").click( function()
+    {
+        var postId = $(this).data("postid");
+        var postType = $(this).data("posttype");
+        var parentId = $(this).data("parentid");
+        
+        $("#flagbox-popup").show();
+        
+        $(".qa-flag-reasons-wrap .closer").click( function()
+        {
+            $("#flagbox-popup").hide();
+        });
+        
+        // focus on first element, then Enter and Escape key work
+        $('.qa-flag-reasons-wrap input').first().focus();
+        
+        $(".qa-go-flag-send-button").click( function()
+        {
+            const flagReason = $("input[name=qa-spam-reason-radio]:checked").val();
+            const flagNotice = $(".qa-spam-reason-text").val();
+            
+            let dataArray = {};
+                dataArray.questionid= flagQuestionid;
```

Comment:
Dlaczego to nie jest zapisane jako literał? I po co te wcięcia? Plus brak spacji przed `=`. Można w niektórych przypadkach zastosować skrócony zapis propertisów: http://es6-features.org/#PropertyShorthand